src/action.js

```diff
@@ -331,6 +331,7 @@ export function dndzone(node, options) {
         dragDisabled = false,
         dropFromOthersDisabled = false,
         dropTargetStyle = DEFAULT_DROP_TARGET_STYLE,
+        transformDraggedElement,
```

Comment:
please also add to the initialisation of config that starts at line 229. 
also please default to an empty function `()=>{}` to prevent issues if we try to invoke it elsewhere and the user didn't provide it

Reply:
Updated